routes/cohortRoute.js

```diff
@@ -0,0 +1,72 @@
+const express = require("express");
+const knex = require("knex");
+
+const router = express.Router();
+
+const knexConfig = require("../knexfile");
+const db = knex(knexConfig.development);
+
+router.get("/", (req, res) => {
+  db("cohorts")
+    .then(cohorts => {
+      res.status(200).json(cohorts);
+    })
+    .catch(err => {
+      res.status(500).json(err);
+    });
+});
+
+router.get("/:id", (req, res) => {
+  const cohortId = req.params.id;
```

Comment:
Strictly a suggestion, using destructuring to pull the id off of req.params. Your implementation is good as is 👍 
```suggestion
  const { cohortId } = req.params;
```